packages/powertools-logger/src/Logger.ts

```diff
@@ -1,13 +1,12 @@
 /**
  * @since 1.0.0
  */
-import { Logger } from "@aws-lambda-powertools/logger";
+import { type Logger, LogLevelThreshold } from "@aws-lambda-powertools/logger";
```

Comment:
`LogLevelThreshold` is exposed from `@aws-lambda-powertools/logger@2.8.0`.

Reply:
I decided to define own `LogLevelThreshold` constant, so we can allow wider range of versions, since v2.0.0